nitro_deps/CMakeLists.txt

```diff
@@ -52,24 +65,37 @@ ExternalProject_Add(
     GIT_REPOSITORY https://github.com/drogonframework/drogon
     GIT_TAG v1.8.6
     CMAKE_ARGS
-    	-DCMAKE_BUILD_TYPE=release
-	-DOPENSSL_USE_STATIC_LIBS=TRUE
-	# -DZLIB_USE_STATIC_LIBS=ON
-	-DBUILD_ORM=OFF
-	-DBUILD_YAML_CONFIG=OFF
-	-DBUILD_EXAMPLES=OFF
-	-DBUILD_CTL=OFF
-	-DCMAKE_EXPORT_COMPILE_COMMANDS=ON
-	-DBUILD_BROTLI=ON
-	-DCMAKE_PREFIX_PATH=${THIRD_PARTY_INSTALL_PATH}
-#	-DCMAKE_FIND_ROOT_PATH=${THIRD_PARTY_INSTALL_PATH} # To set the dir (that will be used to force the look for .a)
+    	-DCMAKE_BUILD_TYPE=Release
```

Comment:
it's release not Release

<img width="697" alt="image" src="https://github.com/janhq/nitro/assets/22268502/8e73890d-c94b-495c-8080-a2b2bf3b9200">